src/lsst/rsp/startup/services/labrunner.py

```diff
@@ -343,14 +450,14 @@ def _merge_aws_creds(self) -> None:
         # information in the container ("original credentials files")
         # is correct, but leave any other user config alone.
         #
-        hc_path = Path(self._env["AWS_SHARED_CREDENTIALS_FILE"])
+        hc_path = Path(self._env.get("AWS_SHARED_CREDENTIALS_FILE", ""))
```

Comment:
It's definitely not safe to continue with a path of `""`. I think that may get interpreted as `.` and you'll then start manipulating the user's home directory or something.